app/scripts/controllers/replicaSet.js

```diff
@@ -79,9 +78,10 @@ angular.module('openshiftConsole')
 
     $scope.deploymentsVersion = APIService.getPreferredVersion('deployments');
     $scope.deploymentConfigsVersion = APIService.getPreferredVersion('deploymentconfigs');
+    $scope.deploymentConfigsLogVersion = APIService.getPreferredVersion('deploymentconfigs/log');
     $scope.eventsVersion = APIService.getPreferredVersion('events');
-    // TODO: update common/constants/apiPreferredVersions for this
-    $scope.deploymentConfigsLogVersion = 'deploymentconfigs/log';
+    $scope.horizontalPodAutoscalersVersion = APIService.getPreferredVersion('horizontalpodautoscalers');
+    $scope.replicationControllersVersion = replicationControllersVersion;
```

Comment:
Just assign it directly to the scope var since that's what we do for other version above.